mix.exs

```diff
@@ -28,10 +28,10 @@ defmodule ExPhoneNumber.Mixfile do
   defp deps do
     [
       {:sweet_xml, "~> 0.6.5"},
-      {:ex_doc, "~> 0.18.0", only: :dev, runtime: false},
+      {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false},
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:ex_spec, "~> 2.0", only: :test},
-      {:excoveralls, "~> 0.10", only: :test},
-      {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false}
+      {:excoveralls, "~> 0.10", only: :test}
```

Comment:
The changes look like done to order the list alphabetically, but they are not as `sweet_xml` is out of order. Either let's put `sweet_xml` last to have alphabetical order or revert this change.